tests/data/wpdb.php

```diff
@@ -14,8 +14,19 @@
 assertType('array<int, mixed>|void|null', wpdb::get_row(null, 'ARRAY_N'));
 
 // wpdb::get_results()
-assertType('stdClass|null', wpdb::get_results());
-assertType('stdClass|null', wpdb::get_results(null, 'OBJECT'));
+    /*
+     *                       Any of ARRAY_A | ARRAY_N | OBJECT | OBJECT_K constants.
```

Comment:
WP does not expect the constants. It expects strings but you can use the corresponding constants.

Reply:
That is a copy of core.
To guide me.